src/content/enterprise/administration/configuration.mdx

```diff
@@ -221,18 +221,43 @@ It is possible to use separate cloud providers for DNS than for storage if neede
 
 ## Okteto Enterprise Components
 
+### backend
```

Comment:
All `env`, `labels` and `annotations` changes are on this PR https://github.com/okteto/docs/pull/130